Arrange so that LLVM's shared object doesn't end up using 'calloc' from
U-Boot's 'malloc_simple.c' when it's initialized.

  https://issues.guix.gnu.org/74270

diff --git a/common/malloc_simple.c b/common/malloc_simple.c
index 5a8ec538f8..4f5e350a3c 100644
--- a/common/malloc_simple.c
+++ b/common/malloc_simple.c
@@ -64,6 +64,8 @@ void *memalign_simple(size_t align, size_t bytes)
 }
 
 #if CONFIG_IS_ENABLED(SYS_MALLOC_SIMPLE)
+void *calloc(size_t nmemb, size_t elem_size)
+  __attribute__ ((visibility ("hidden")));
 void *calloc(size_t nmemb, size_t elem_size)
 {
 	size_t size = nmemb * elem_size;
